README.md

```diff
@@ -17,16 +17,45 @@ NOTE: The dist folder is autogenerated code based off of what is in the src fold
 
 Highlighting needs to be given raw content (text, including bolding and links). See the example from the Carleton page.
 
+#### Highlighting without context 
+
 ```javascript
-const persistent = [
+const changes = [
     "ranked #1 in Undergraduate Teaching by U.S. News & World Report for over a decade",
     "Founded in 1866",
     "Admissions is highly selective",
     "Carleton is one of the highest sources of undergraduate students pursuing doctorates",
 ];
-var arrayLength = persistent.length;
-for (var i = 0; i < arrayLength; i++) {
-    highlightPersistentContent(persistent[i], "#99FF84");
+let arrayLength = changes.length;
+for (let i = 0; i < arrayLength; i++) {
+    highlightContent(changes[i], "#99FF84");
+}
+```
+
+#### Highlighting with context 
+
+```javascript
+const arr = [
+    {
+        "content_before": "college ",
+        "highlight": "have",
+        "content_after": " included"
+    },
+    {
+        "content_before": "Admissions ",
+        "highlight": "is highly selective",
+        "content_after": " with an acceptance rate of 16.5% in 2022"
+    },
+    {
+        "content_before": "founded ",
+        "highlight": "in",
+        "content_after": " 1866"
+    },
+];
+
+let arrayLength = arr.length;
+for (let i = 0; i < arrayLength; i++) {
+    highlightContentWithContext(arr[i], "#99FF84");
```

Comment:
Say if we are using this (only care about the text), how much of the page will actually be highlighted with this? (how much of the page is actually pure text and no weird HTML formatting?)

Reply:
I would say 60-80%? if the highlighted part is a link or the context is part of one, it won't work, but any other case should be fine